broker/src/main/java/io/zeebe/broker/system/partitions/ZeebePartition.java

```diff
@@ -345,9 +351,13 @@ public void onDiskSpaceAvailable() {
   public void resumeProcessing() {
     actor.call(
         () -> {
-          context.setProcessingPaused(false);
-          if (context.getStreamProcessor() != null && context.shouldProcess()) {
-            context.getStreamProcessor().resumeProcessing();
+          try {
+            context.setProcessingPaused(false);
+            if (context.getStreamProcessor() != null && context.shouldProcess()) {
+              context.getStreamProcessor().resumeProcessing();
+            }
+          } catch (final IOException e) {
```

Comment:
ioe?

Reply:
What happens then ? Do we return an error?

Reply:
Same as above. The api to resume processing is async, so the error is not returned to the client.